src/components/DetailsNavBar/Evolution/EvolutionChain.tsx

```diff
@@ -0,0 +1,61 @@
+//@ts-nocheck
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { LazyLoadImage } from "react-lazy-load-image-component";
+import { pokemonEvolutionChain, pokemonRoute } from "../../../utils/baseUrl";
+import { useAppSelector } from "../../../redux/hook";
+
+const EvolutionChain = ({ evolutionId }) => {
+  const [evolutionChain, setEvolutionChain] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { currentPokemon } = useAppSelector(({ pokemon }) => pokemon);
+
+  useEffect(() => {
+    const fetchEvolutionChain = async () => {
+      try {
+        const { data } = await axios.get(
+          `${pokemonEvolutionChain}/${evolutionId}/`
+        );
+        const species = data.chain.species.name;
+        const evolvesTo = data.chain.evolves_to.map(
```

Comment:
Destructuring would make it easier to use

```suggestion
       const { species, evolves_to } = data.chain;
       
        const species = species.name;
        const evolvesTo = evolves_to.map(
```